integrated_channels/blackboard/migrations/0001_initial_squashed_0011_alter_blackboardlearnerassessmentdatatransmissionaudit_enterprise_course_enrollment_id.py

```diff
@@ -0,0 +1,129 @@
+    BlackboardEnterpriseCustomerConfiguration = apps.get_model('blackboard', 'BlackboardEnterpriseCustomerConfiguration')
+    for config in BlackboardEnterpriseCustomerConfiguration.objects.all():
+        config.uuid = uuid.uuid4()
+        config.save()
+
+
+class Migration(migrations.Migration):
+
+    replaces = [('blackboard', '0001_initial'), ('blackboard', '0002_auto_20200930_1723'), ('blackboard', '0003_blackboardlearnerdatatransmissionaudit'), ('blackboard', '0004_blackboard_tx_chunk_size_default_1'), ('blackboard', '0005_blackboardlearnerassessmentdatatransmissionaudit'), ('blackboard', '0006_auto_20210708_1446'), ('blackboard', '0007_auto_20210909_1536'), ('blackboard', '0008_auto_20210923_1727'), ('blackboard', '0009_alter_blackboardenterprisecustomerconfiguration_enterprise_customer'), ('blackboard', '0010_auto_20211221_1532'), ('blackboard', '0011_alter_blackboardlearnerassessmentdatatransmissionaudit_enterprise_course_enrollment_id')]
+
+    dependencies = [
+        ('enterprise', '0151_add_is_active_to_invite_key'),
+        ('enterprise', '0113_auto_20200914_2054'),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='BlackboardLearnerDataTransmissionAudit',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('blackboard_user_email', models.EmailField(help_text='The learner`s Blackboard email. This must match the email on edX in order for both learner and content metadata integrations.', max_length=255)),
+                ('completed_timestamp', models.CharField(help_text='Represents the Blackboard representation of a timestamp: yyyy-mm-dd, which is always 10 characters.', max_length=10)),
+                ('course_id', models.CharField(max_length=255)),
+                ('course_completed', models.BooleanField(default=True, help_text="The learner's course completion status transmitted to Blackboard.")),
+                ('enterprise_course_enrollment_id', models.PositiveIntegerField(db_index=True)),
+                ('grade', models.DecimalField(blank=True, decimal_places=2, max_digits=3, null=True)),
+                ('total_hours', models.FloatField(blank=True, null=True)),
+                ('created', models.DateTimeField(auto_now_add=True)),
+                ('error_message', models.TextField(blank=True)),
+                ('status', models.CharField(max_length=100)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='BlackboardLearnerAssessmentDataTransmissionAudit',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('blackboard_user_email', models.CharField(max_length=255)),
+                ('enterprise_course_enrollment_id', models.IntegerField(db_index=True)),
```

Comment:
Changed this from `Positive to Integer`.
Remove alter migrations code from bottom.